lib/src/third_party/boringssl/generated_bindings.dart

```diff
@@ -3539,56 +3211,59 @@ class crypto_mutex_st extends ffi.Union {
   external ffi.Array<ffi.Uint8> padding;
 }
 
-class ctr_drbg_state_st extends ffi.Opaque {}
-
 class dh_st extends ffi.Opaque {}
 
-class div_t extends ffi.Struct {
-  @ffi.Int()
-  external int quot;
-
-  @ffi.Int()
-  external int rem;
-}
-
-class drand48_data extends ffi.Struct {
-  @ffi.Array.multi([3])
-  external ffi.Array<ffi.UnsignedShort> __x;
-
-  @ffi.Array.multi([3])
-  external ffi.Array<ffi.UnsignedShort> __old_x;
-
-  @ffi.UnsignedShort()
-  external int __c;
-
-  @ffi.UnsignedShort()
-  external int __init;
-
-  @ffi.UnsignedLongLong()
-  external int __a;
-}
-
 class dsa_st extends ffi.Opaque {}
 
 class ec_group_st extends ffi.Opaque {}
 
 class ec_key_st extends ffi.Opaque {}
 
-class ec_method_st extends ffi.Opaque {}
-
 class ec_point_st extends ffi.Opaque {}
 
-class ecdsa_method_st extends ffi.Opaque {}
+/// Low-level signing and verification.
+///
+/// Low-level functions handle signatures as |ECDSA_SIG| structures which allow
+/// the two values in an ECDSA signature to be handled separately.
+class ecdsa_sig_st extends ffi.Struct {
+  external ffi.Pointer<BIGNUM> r;
 
-class ecdsa_sig_st extends ffi.Opaque {}
+  external ffi.Pointer<BIGNUM> s;
+}
 
 class engine_st extends ffi.Opaque {}
 
-class env_md_ctx_st extends ffi.Opaque {}
+class env_md_ctx_st extends ffi.Struct {
+  /// digest is the underlying digest function, or NULL if not set.
+  external ffi.Pointer<EVP_MD> digest;
+
+  /// md_data points to a block of memory that contains the hash-specific
+  /// context.
+  external ffi.Pointer<ffi.Void> md_data;
+
+  /// pctx is an opaque (at this layer) pointer to additional context that
+  /// EVP_PKEY functions may store in this object.
+  external ffi.Pointer<EVP_PKEY_CTX> pctx;
+
+  /// pctx_ops, if not NULL, points to a vtable that contains functions to
+  /// manipulate |pctx|.
+  external ffi.Pointer<evp_md_pctx_ops> pctx_ops;
+}
```

Comment:
This became non-opaque... I'm not sure why?

Since I think I asked for dependencies to be opaque, and only listed `cbs_st` and `cbb_st` in `includes`.
But it seems to make have everything included by dependency as non-opaque.

It's not a huge issue.

Reply:
cc @mannprerak2

Reply:
Is it possible that this struct is being referred to by Value somewhere? Since opaque only works with pointer references. (I do see its usage by value via a typedef reference) 

Maybe compare the before and after logs for v 7.0.0 and v 7.1.0
For the older version, logs should contain warnings like - "skipped <declaration> due to incomplete member reference"
But since 7.1.0 they are now complete (as we can now find definitions across entry-points).

Reply:
Another case I found is when something like this happens - 

```C
void func(struct A at, struct OpaqueByIntention to);
```
Suppose this is the only place where `struct A` is used by value, then generated A will not be Opaque, even though `func` (that used A by value) is skipped due to `OpaqueByIntention` pass by value. 
This happens because of the way we parse our code, if the config is set to `dependency-only: opaque` we immediately parse the dependencies if the struct at any point has been referred by value.